### PR TITLE
Including libft

### PR DESCRIPTION
**Pull request by FernandaMatt**
libft from fcaetno's fdf

### CODE REVIEW

File: libft/ft_atoi.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+static int	ft_catnbr(char *str)
+{
+	unsigned int	c;
+	unsigned int	mult;
+	int				result;
+
+	c = 0;
+	while (str[c] >= '0' && str[c] <= '9' && str[c] != 0)
+			c++;
```
Comment (michaelrodriguess) [inline libft/ft_atoi.c:28]: Wrong indentation.

**Review by michaelrodriguess — CHANGES_REQUESTED** [review]
Fix indentations in file ft_atoi.c at line 28

File: libft/ft_atoi.c (hunk 1 above)
Comment (FernandaMatt) [reply libft/ft_atoi.c:28]: Ok, will work on it! (:

Comment (FernandaMatt) [reply libft/ft_atoi.c]: Done! Also fixed some norminette erros in main.c

**Review by michaelrodriguess — APPROVED** [review]

**Pull request merged**